ruskeyverbs/views.py

```diff
@@ -1,14 +1,39 @@
 from itertools import chain
 from django.shortcuts import render
-from django.http import HttpResponse
+from django.http import HttpResponse, HttpResponseRedirect, Http404
 from django.contrib.auth.decorators import login_required
 from django.contrib.auth.mixins import LoginRequiredMixin
+from django.contrib import messages
 from django.views import generic
+from django.core.exceptions import ObjectDoesNotExist
 from django.db.models import Min
 from django.db.models import Q
+from django.shortcuts import get_object_or_404
+from django.urls import reverse
+from django.core import serializers
+from fuzzywuzzy import fuzz
+from random import shuffle, randint
 from .models import Verb, Example, PerformancePerExample
+from .forms import FillInTheBlankForm, ArrangeWordsForm, ReproduceSentenceForm, MultipleChoiceForm
+import datetime
+import decimal
 
-# Create your views here.
+
+def strip_punct_lower(some_string, stressed=False):
```

Comment:
Not that big a deal, but a regex would be more efficient here.

Reply:
Swapped in a regex.